### PR TITLE
chore: pin all dependencies

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -417,6 +417,15 @@
     debug "^4.1.0"
     globals "^11.1.0"
 
+"@babel/types@7.21.4", "@babel/types@^7.21.4":
```
Comment (ansgarm) [inline yarn.lock:420]: Quick question: The `@babel/types@^7.21.4` one (with the caret) is a transitive dependency, right? So basically a dependency of ours using that version constraint?

**Review by ansgarm — APPROVED** [review]
Looks good (if CI passes 😄), just one small non-blocking question

File: yarn.lock (hunk 1 above)
Comment (DanielMSchmidt) [reply yarn.lock:420]: I think so, yeah :)

File: packages/@cdktf/provider-schema/src/__tests__/provider-schema.test.ts [hunk 2]
```diff
@@ -11,17 +12,12 @@ import {
...
-    if (typeof value !== "object" || value === null) return false;
-
-    return value["format_version"] !== undefined;
-  },
-  print: (value: any) => {
-    value["format_version"] = "STUBBED VERSION";
-    return JSON.stringify(value, null, 2);
-  },
-});
+function stubVersion(value: any) {
```
Comment (ansgarm) [inline packages/@cdktf/provider-schema/src/__tests__/provider-schema.test.ts:15]: I'd call this something like "clean" or "sanitize" 🤔

**Pull request merged**